uber/templates/preregistration/preregbase.html

```diff
@@ -18,7 +15,7 @@
     };
     setInterval(checkCap, 1000 * 1 * 90);
 
-{% if COLLECT_FULL_ADDRESS %}
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
Note to self: this is pretty unnecessary now that you can disable validation and hide fields via plugin.